src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -157,7 +160,7 @@ public ActivityTraceFlags TraceFlags
     /// <summary>
     /// Gets or sets the log <see cref="Microsoft.Extensions.Logging.LogLevel"/>.
     /// </summary>
-    // todo: [Obsolete("Use Severity instead LogLevel will be removed in a future version.")]
+    [Obsolete("Use Severity instead LogLevel will be removed in a future version.")]
```

Comment:
```suggestion
    [Obsolete("Use Severity instead LogLevel will be removed in a future release with a major version.")]
```

Reply:
I think a bit more discussion is needed around our strategy on this and then we can land a policy doc + update all the warnings we have sprinkled around?

/cc @alanwest